src/components/message/MessageText.jsx

```diff
@@ -1,26 +1,40 @@
 import React from "react";
+import connect from "react-redux/es/connect/connect";
 import parseEmoticons from "./parsers/parseEmoticons";
 import parseMedia from "./parsers/parseMedia";
 import parseFormatting from "./parsers/parseFormatting";
 import styled from "styled-components";
+import theme from "../../constants/theme";
 
 const MessageTextContainer = styled.div`
 	.emoticon {
 		max-height: 24px;
 	}
+	&.mention {
+		background-color: ${theme.mentionBackgroundColor ? theme.mentionBackgroundColor : '#faf2cc'};
+		color: ${theme.mentionForegroundColor}
+	}
 `;
 
+const mapStateToProps = (state, ownProps) => ({
+	nick: state.users[state.userSettings.user].nick
```

Comment:
The user id only exists on the userSettings which is strange.

Reply:
Yeah you can fix that. Right now the init data includes an object called user, but we're not consuming it in the user.js reducer.